### PR TITLE
uwufetch: 2.0 -> 2.1

### PR DESCRIPTION
**Pull request by wineee**
###### Things done

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.05 Release Notes (or backporting 22.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2305-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/tools/misc/uwufetch/default.nix [hunk 1]
```diff
@@ -1,20 +1,25 @@
...
-    ./fix-paths.patch
-  ];
+  postPatch = ''
+    substituteInPlace uwufetch.c \
+      --replace "/usr/lib/uwufetch" "$out/lib/uwufetch" \
+      --replace "/usr/local/lib/uwufetch" "$out/lib/uwufetch" \
+      --replace "/etc/uwufetch/config" "$out/etc/uwufetch/config"
+    # fix command_path for package manager (nix-store)
+    substituteInPlace fetch.c \
+      --replace "/usr/bin" "/run/current-system/sw/bin"
```
Comment (lourkeur) [inline pkgs/tools/misc/uwufetch/default.nix:21]: for darwin, the `PKGPATH` is `"/usr/local/bin"`. Do we know what the proper substitute would be?

**Review by bbjubjub2494 — APPROVED** [review]
Result of `nixpkgs-review pr 216340` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>uwufetch</li>
  </ul>
</details>

`strings` clean, functionality looks good.

**Comment by equirosa** [comment]
Result of `nixpkgs-review pr 216340` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>uwufetch</li>
  </ul>
</details>

**Comment by equirosa** [comment]
Result of `nixpkgs-review pr 216340` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>uwufetch</li>
  </ul>
</details>

File: pkgs/tools/misc/uwufetch/default.nix (hunk 1 above)
Comment (wineee) [reply pkgs/tools/misc/uwufetch/default.nix:21]: `/usr/local/bin/brew`,  I think it's fine to keep this path

**Comment by SuperSandro2000** [comment]
Darwin fails

```
Builder called die: Cannot wrap '/nix/store/9kfz63d34d9nwpmz8d2arjhjmwcjgr0w-uwufetch-2.1/bin/uwufetch' because it is not an executable file
```

**Review by bbjubjub2494 — APPROVED** [review]

**Comment by lourkeur** [comment]
> Darwin fails
> 
> ```
> Builder called die: Cannot wrap '/nix/store/9kfz63d34d9nwpmz8d2arjhjmwcjgr0w-uwufetch-2.1/bin/uwufetch' because it is not an executable file
> ```

Can you check if `uwufetch` might be in `$out/local/bin/uwufetch`? I'm suspecting the hand-made Makefile.

**Pull request merged**